mod/wikirate/set/type/claim.rb

```diff
@@ -26,9 +26,13 @@ def indirect_contributor_search_args
     #rename "name" to "Claim"
     #add a div for claim word counting
     %{
-      #{ formgroup 'Note', raw( name_field form ), :editor=>'name', :help=>true }
-      <div class='note-counting'>
-        <span class='note-counting-number'>100</span> character(s) left
+      <div class='row'>
+        <div class='col-md-12'>
+            #{ formgroup 'Note', raw( name_field form ), :editor=>'name', :help=>true }
```

Comment:
Line is too long. [87/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.<br>Space inside parentheses detected.<br>Space inside string interpolation detected.